fluent-bundle/src/resolver.ts

```diff
@@ -111,7 +111,7 @@ function getArguments(
   args: Array<Expression | NamedArgument>
 ): Arguments {
   const positional: Array<FluentValue> = [];
-  const named: Record<string, FluentValue> = Object.create(null);
```

Comment:
Question: Were these new errors?

Reply:
Yes. `Object.create()` returns `any`, and the `@typescript-eslint/recommended` set includes [no-unsafe-assignment](https://github.com/typescript-eslint/typescript-eslint/blob/v4.28.4/packages/eslint-plugin/docs/rules/no-unsafe-assignment.md) which complains about assigning that to a more specific target.

So by moving the typing to an `as`, we do the same thing as before but ESLint stops complaining.